run_desktop_tests.sh

```diff
@@ -86,4 +87,10 @@ then
 
   echo "Running unit tests on the desktop (debug mode + CTAP1)..."
   cargo test --features std,with_ctap1
+
+  echo "Running unit tests on the desktop (release mode + CTAP2.1)..."
+  cargo test --release --features std,with_ctap2_1
+
+  echo "Running unit tests on the desktop (debug mode + CTAP2.1)..."
+  cargo test --features std,with_ctap2_1
```

Comment:
So we never test `with_ctap1` + `with_ctap2_1`? Having both is a relevant use case, and may cause issues like exceeding available memory, so I'd test that too.

Reply:
Tests are added.